### DISCUSSION TITLE
Is it possible to launch an application on a existing remote session from the client command line?

### DISCUSSION THREAD

**Discussion by jzyrobert** [Xpra-org/xpra#3783] [Q&A] [answered]
So the setup is:
Remote: Ubuntu 22 running Xpra 4.4.3, with `xpra start :100`
Client: Windows 10 running Xpra 4.4.3, using `Xpra attach ssh://<host>/100` to connect

Then, how can I start an application (e.g. xterm) from the windows client without restarting the connection?
If I run `Xpra attach ssh://host/100 --start=xterm` again, it works but restarts the connection so all existing windows are reloaded and get moved
I could either:
- Start xterm from the xpra system tray
- Open a SSH terminal with `$DISPLAY=100` set and run `xterm` there

But I would prefer if I could do it from Windows powershell directly, basically a command line equivalent of launching something from the system tray.
I've looked through the `man xpra` documentation as well as the `docs/` folder and the Wiki but could not see any obvious command to achieve this.

Thanks

**Comment by totaam** [comment, marked as answer]
From your xpra system tray, there should be a start menu.
With versions older than 4.4.4 (not yet released), you may need this patch on Ubuntu: https://github.com/Xpra-org/xpra/commit/a42e2343ee572ff2edb28ece0b38904969c75470

Alternatively, you can use `xpra control`:
* directly on the server: `xpra control :100 start xterm`
* from your client: `Xpra_cmd control ssh://host/100 start xterm`

**Reply by jzyrobert** [reply]
`control start` is exactly what I am looking for, thanks!
